docs/4.2/admin-guide.md

```diff
@@ -188,8 +188,10 @@ Docs](cli-docs.md#teleport-start) or run `teleport start --help`
 
 ### Configuration File
 
-Teleport uses the YAML file format for configuration. A sample configuration
-file is shown below. By default, it is stored in `/etc/teleport.yaml`
+Teleport uses the YAML file format for configuration. A sample configuration file is shown below. By default, it is stored in `/etc/teleport.yaml`. 
```

Comment:
One of the core issues with this sentence, is that we say  `a sample configuration file`. What we have below is really a reference.  We need to split this up, ( not necessarily in this PR ) but I'll DM you another project I'm working on @travelton

Reply:
Agreed. It's information overload. I'm hoping the inclusion of the sample configurations lead folks to the simplified samples.